docs/docsite/rst/roadmap/ROADMAP_2_5.rst

```diff
@@ -0,0 +1,218 @@
+============================
+Ansible Project, 2.5
+============================
+**Core Engine Freeze and Module Freeze: 15 January 2018**
+
+**Core and Curated Module Freeze: 15 January 2018**
+
+**Community Module Freeze: 31 January 2018**
+
+**Release Candidate 1 will be 14 February, 2018**
+
+**Target: March 2018**
+
+.. contents:: Topics
+
+Engine improvements
+-------------------
+- Polish and consolidate CLI
+
+  - Add a debug flag that prints log entries to stdout
+  - Investigate implementing subcommands
+- Clean up output formatting
+- Reimplement task result `invocation` injection control-side, but only when module didn't return it.
+
+  - always use heuristic sensitive-value filtering on injected values
+  - if heuristic filter triggers, add a warning that perhaps use of task-level no_log should be considered
+  - consider skipping injection for Python module type (even if it's missing, eg catastrophic module failure) to prevent potentially injecting a value that was arg-level no_log'd on the target side.
+  - consider addition of NO_INVOCATION config
+- Automatic shared SSH tunneling support for networked connection plugins (dynamic jumphost for SSH/WinRM/whatever)
```

Comment:
This should've been stretch- I didn't move it in time in the 2.5 project

Reply:
Could I get a review again, @nitzmahone.  I removed that item.